web/src/components/shared/ConfigureGraphs.jsx

```diff
@@ -1,54 +1,73 @@
 import * as React from "react";
 
-class ConfigureGraphs extends React.Component {
-  render() {
-    const {
-      toggleConfigureGraphs,
-      updatePromValue,
-      promValue,
-      savingPromValue,
-      savingPromError,
-      onPromValueChange
-    } = this.props;
-
-    return (
-      <div className={`${toggleConfigureGraphs ? "Modal-body" : "ConfigureGraphs--wrapper"} flex-column flex1`}>
-        {toggleConfigureGraphs && <h2 className="u-fontSize--largest u-fontWeight--bold u-textColor--primary u-marginBottom--10">Configure graphs</h2>}
-        <p className="u-fontSize--normal u-textColor--bodyCopy u-lineHeight--normal u-marginBottom--20">To see graphs and metrics, provide the address of your Prometheus installation.<br />This must be resolvable from the Admin Console installation.</p>
-        <h3 className="u-fontSize--normal u-fontWeight--bold u-textColor--primary u-marginBottom--10">Prometheus endpoint</h3>
-        <div className="EditWatchForm flex-column">
-          <div className="flex alignItems--center">
-            <input
-              type="text"
-              className="Input u-marginRight--10"
-              placeholder="https://prometheus-k8s.default.svc.cluster.local:9090"
-              value={promValue}
-              onChange={onPromValueChange}
-            />
+const ConfigureGraphs = ({
+  toggleConfigureGraphs,
+  updatePromValue,
+  promValue,
+  savingPromValue,
+  savingPromError,
+  onPromValueChange,
+  placeholder,
+}) => {
+  return (
+    <div
+      className={`${
+        toggleConfigureGraphs ? "Modal-body" : "ConfigureGraphs--wrapper"
+      } flex-column flex1`}
+    >
+      {toggleConfigureGraphs && (
+        <h2 className="u-fontSize--largest u-fontWeight--bold u-textColor--primary u-marginBottom--10">
+          Configure graphs
+        </h2>
+      )}
+      <p className="u-fontSize--normal u-textColor--bodyCopy u-lineHeight--normal u-marginBottom--20">
+        To see graphs and metrics, provide the address of your Prometheus
+        installation.
+        <br />
+        This must be resolvable from the Admin Console installation.
+      </p>
+      <h3 className="u-fontSize--normal u-fontWeight--bold u-textColor--primary u-marginBottom--10">
+        Prometheus endpoint
+      </h3>
+      <div className="EditWatchForm flex-column">
+        <div className="flex alignItems--center">
+          <input
+            type="text"
+            className="Input u-marginRight--10"
+            placeholder={
+              placeholder ||
```

Comment:
Again, this file was linted to make it prettier. This line, adding the placeholder, is the only major change.